docs/sources/userguide/dockervolumes.md

```diff
@@ -141,6 +146,14 @@ be deleted.  To delete the volume from disk, you must explicitly call
 `docker rm -v` against the last container with a reference to the volume. This
 allows you to upgrade, or effectively migrate data volumes between containers.
 
+> **Note:** Docker will not warn you when removing a container *without* 
+> providing the `-v` option to delete its volumes. If you remove containers
+> without using the `-v` option, you may end up with "orphaned" volumes; 
+> volumes that are no longer referenced by a container.
+> Orphaned volumes are difficult to get rid off and can take up a large amount
+> of disk space. We're working on improving volume management and you can check
+> progress on this in [pull request #8484](https://github.com/docker/docker/pull/8484)
```

Comment:
Are there any workarounds or any way to mitigate the issue of orphaned volumes? Worth mentioning here, if so.